A3.md

```diff
@@ -165,21 +175,27 @@ message TraceNode {
 message SubchannelTrace {
```

Comment:
All of the fields in this message apply to both the parent channel and to each subchannel.  So how about something like this:

```
message TraceData {
  int32 num_nodes_logged = 1;
  google.protobuf.Timestamp start_time = 2;
  repeated TraceNode nodes = 3;
}

message ChannelTrace {
  TraceData data = 1;  // parent channel data
  int32 num_subchannels_seen = 2;
  repeated TraceData subchannel_data = 3;
}
```